.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,27 @@
+name: CIFuzz
+on:
+  push:
+    branches:
+      - master
```

Comment:
@nathannaveen I think it might make sense to trigger cifuzz for pull_request instead (so commit can be fixed before the merge)?
```
  pull_request:
    branches:
      - master
```

Reply:
I have done that, thanks!